<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QMainWindow" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="inputFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="openVideoFileButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Video File...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="VideoWidget" name="processedFrame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="hueSlider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>350</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHue">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHueTolerance">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="hueTolerance">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerance</string>
    </property>
   </widget>
   <widget class="QSlider" name="saturationSlider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>207</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSaturation">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QLabel</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
